sources.cfg

```diff
@@ -3,3 +3,5 @@ extends = http://plonesource.org/sources.cfg
 extensions = mr.developer
 
 auto-checkout =
+# https://github.com/4teamwork/ftw.pdfgenerator/pull/44
+    ftw.pdfgenerator
```

Comment:
Since this change does not work with older ftw.pdfgenerator versions, we should probable add a version constraint.
@shylux can you
- cut a release of `ftw.pdfgenerator`
- add a version constraint of the pdfgenerator version or never to the book's `setup.py`
- remove from auto-checkout